common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -968,7 +969,13 @@ def get_answer(self, _data):
                 new_answer = {answer_id: answers[answer_id]}
             new_answers.update(new_answer)
 
-        return {'answers': new_answers}
+        return {
+            'answers': new_answers,
+            'correct_status_html': self.runtime.render_template(
```

Comment:
Could you explain again why this was needed? It seems to differ from how the status_span.html template is used elsewhere.

A comment may also be needed to document the decision.